runelite-client/src/main/java/net/runelite/client/plugins/npchighlight/NpcIndicatorsPlugin.java

```diff
@@ -85,6 +88,8 @@
 	private static final String TAG = "Tag";
 	private static final String UNTAG = "Un-tag";
 
+	private static Pattern COLOR_REGEX = Pattern.compile("<col=([0-9a-f]+)>");
```

Comment:
This should be `final` since it won't ever change.
```suggestion
	private static final Pattern COLOR_REGEX = Pattern.compile("<col=([0-9a-f]+)>");
```